modules/cnv-vm-wizard-fields-web.adoc

```diff
@@ -42,37 +34,43 @@ endif::[]
 |Container
 |Provision virtual machine from a bootable operating system container located in a registry accessible from the cluster. Example: `_kubevirt/cirros-registry-disk-demo_`.
 
-|Cloned Disk
-|Provision source is a cloned disk.
+|Disk
+|Provision virtual machine from a disk.
 
-|Import
-|Import virtual machine from a supported provider.
+|Attach disk
+|
+|Provision an existing disk that was previously cloned or created and made available in the Persistent Volume Claims. When this option is selected, the operating system, flavor, and workload profile fields must be filled manually.
```

Comment:
The first sentence ("Provision an existing disk...") is a little confusing/long. I suggest rewriting it for clarity.

The second sentence needs formatting around the UI elements, which should also be capitalized (for example, `*Operating System*`) to match the UI. Please also rewrite it to not use the passive voice (Suggestion: `When this option is selected, you must manually complete the *Operating System*, *Flavor*, and *Workload Profile* fields.`)

Reply:
Modified the sentence:
"Attach an existing disk..."
UI elements have been capitalized and bolded.
Passive sentence changed to active tense.